test/common.py

```diff
@@ -56,6 +56,7 @@
 EMTEST_ALL_ENGINES = None
 EMTEST_SKIP_SLOW = None
 EMTEST_SKIP_FLAKY = None
+EMTEST_RETRY_FLAKY = None
```

Comment:
Just a nitpick, but if we are adding a separate env variable anyway, I'm tempted to say it should contain number of retries instead of hardcoding one. As in, if we read it anyway, might as well put a useful value there.

Reply:
Sounds good!

Reply:
Tindakan Node.js 16 tidak digunakan lagi. Harap perbarui tindakan berikut untuk - [ ] menggunakan Node.js 20: